app/models/field.rb

```diff
@@ -0,0 +1,19 @@
+class Field < ApplicationRecord
+  belongs_to :calculator
+  before_save :set_selector
```

Comment:
change to `before_create`. it needs to be set only once

Reply:
Thanks. I've changed it.